test/python/providers/test_backend_v2.py

```diff
@@ -0,0 +1,68 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=missing-class-docstring,missing-function-docstring
+# pylint: disable=missing-module-docstring
```

Comment:
I feel like we _shouldn't_ do this, but also, tests are never going to be in our API documentation...

Reply:
Yeah, I just get grumpy that pylint tries to force pointless docstrings for everything in a test module when they provide no value (what does a class doctring ```"""Tests for Target"""` add on top of a class named `TestTarget`). Especially as a lot of test runners replace a test id (ie test method) with the docstring in verbose modes. You can get the same value if you name your modules, classes, and methods verbosely with a useful name that describes what's under test.

Reply:
Perhaps it'd be nice if we set up the `tox` job to run the linter on `test` with these additional suppressions?  A separate PR for sure, but I generally agree.  I personally use the method docstrings to provide more explanation for the test, but it doesn't need to be enforced for everything.

Reply:
Yeah, I think that's a good idea. I've been meaning to do that for a long time and always seem to forget